lib/Service/ActorService.php

```diff
@@ -287,14 +286,14 @@ public function cacheLocalActorByUsername(string $username, bool $refresh = fals
 	 */
 	private function updateCacheLocalActorName(Person &$actor) {
 		$user = $this->userManager->get($actor->getUserId());
-		$account = $this->accountManager->getAccount($user);
 
 		try {
+			$account = $this->accountManager->getAccount($user);
 			$displayNameProperty = $account->getProperty(IAccountManager::PROPERTY_DISPLAYNAME);
 			if ($displayNameProperty->getScope() === IAccountManager::VISIBILITY_PUBLIC) {
 				$actor->setName($displayNameProperty->getValue());
 			}
-		} catch (PropertyDoesNotExistException $e) {
+		} catch (Exception $e) {
```

Comment:
I do not know when this function is used, but maybe some feedback that update failed is nice?

Reply:
The function is needed when we refresh the cache for local social accounts. It check the displayname in the settings of the nextcloud account. 

I have an issue with NC15 that I will investigate, but you're right I will add some logs in here